### ISSUE TITLE
Missing map files cause error during initial editor launch

### ISSUE THREAD

**Issue by starikcetin** [starikcetin/Eflatun.SceneReference#88]
Steps to reproduce:
1. Close Unity.
1. Remove the map files.
1. Launch project.

Error message:
```
[Eflatun.SceneReference] Scene GUID to path map file not found!
UnityEngine.Debug:LogError (object)
Eflatun.SceneReference.Logger:Error (string) (at ./Packages/com.eflatun.scenereference/Runtime/Logger.cs:7)
Eflatun.SceneReference.SceneGuidToPathMapProvider:Load () (at ./Packages/com.eflatun.scenereference/Runtime/SceneGuidToPathMapProvider.cs:66)
Eflatun.SceneReference.SceneGuidToPathMapProvider:LoadIfNotAlready () (at ./Packages/com.eflatun.scenereference/Runtime/SceneGuidToPathMapProvider.cs:55)
Eflatun.SceneReference.SceneGuidToPathMapProvider:get_SceneGuidToPathMap () (at ./Packages/com.eflatun.scenereference/Runtime/SceneGuidToPathMapProvider.cs:27)
Eflatun.SceneReference.Editor.MapGeneratorTriggers.SceneAssetPostprocessor:OnPostprocessAllAssets (string[],string[],string[],string[]) (at ./Packages/com.eflatun.scenereference/Editor/MapGeneratorTriggers/SceneAssetPostprocessor.cs:11)
UnityEditor.AssetPostprocessingInternal:PostprocessAllAssets (string[],string[],string[],string[],string[],bool)
```

The map files are expected to be missing during the initial launch, so we should not print any errors at that point. Though the map generation still turns out fine, I am still marking this issue as a bug since it looks like one.

**Comment by starikcetin** [comment]
Fixed in #89. Will be included in the next release.

**Comment by starikcetin** [comment]
Included in [3.2.1](https://github.com/starikcetin/Eflatun.SceneReference/releases/tag/3.2.1).
